src/Test/L0/Worker/SaveStateFileCommandL0.cs

```diff
@@ -311,15 +311,8 @@ public void SaveStateFileCommand_Heredoc_MissingNewLine()
             using (var hostContext = Setup())
             {
                 var stateFile = Path.Combine(_rootDirectory, "heredoc");
-                var content = new List<string>
-                {
-                    "MY_STATE<<EOF",
-                    "line one",
-                    "line two",
-                    "line three",
-                    "EOF",
-                };
-                WriteContent(stateFile, content, " ");
+                string content = "MY_OUTPUT<<EOF line one line two line three EOF";
+                TestUtil.WriteContent(stateFile, content);
                 var ex = Assert.Throws<Exception>(() => _saveStateFileCommand.ProcessCommand(_executionContext.Object, stateFile, null));
                 Assert.Contains("Matching delimiter not found", ex.Message);
```

Comment:
As I mentioned earlier, it wasn't easy to spot the difference between this test case and the one that begins on line 190 and I was puzzling over why that one is expected to succeed and this one is expected to fail.

The difference is just the third arg (`" "`) passed to `WriteContent` which "tricks" `WriteContent` into mashing all the content together on a single line.

I think it's clearer to just be explicit about that:

```
string content = "MY_OUTPUT<<EOF line one line two line three EOF";
```